src/openrpc/metrics.json

```diff
@@ -1130,6 +1268,16 @@
           }
         ]
       },
+      "MetricOptions": {
+        "title": "MetricOptions",
+        "description": "Additional options, tags, or labels to include with a metrics call.",
+        "type": "object",
+        "properties": {
+          "agePolicy": {
```

Comment:
Age Policy doesnt seem to be a required field for MetricOptions. So are we says apps can call metricOptions with empty?